src/content/tutorials/fleet-management/deletion-prevention/index.md

```diff
@@ -0,0 +1,153 @@
+
+## Adding deletion prevention to an existing cluster
+
+Our CAPI-based cluster charts allow setting the option `global.metadata.preventDeletion=true`. It can be toggled in the values of the chart:
+
+```yaml
+---
+apiVersion: v1
+data:
+  values: |
+    global:
+      connectivity:
+        availabilityZoneUsageLimit: 3
+        network: {}
+        topology: {}
+      controlPlane: {}
+      metadata:
+        description: My test cluster
+        name: mycluster
+        organization: myorg
+
+        ###############################################
+        preventDeletion: false # <==== set this to true
+        ###############################################
+      nodePools:
+        # [...]
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  labels:
+    giantswarm.io/cluster: mycluster
+  name: mycluster-userconfig
+  namespace: org-myorg
+---
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+  name: mycluster
+  namespace: org-myorg
+spec:
+  # [...]
+  name: cluster-aws
+  namespace: org-myorg
+  userConfig:
+    configMap:
+      name: mycluster-userconfig
+      namespace: org-myorg
+  version: ""
+```
+
+After applying the config map change on the management cluster, you should see the deletion prevention label on certain resources such as the `Cluster` object.
+
+## Creating a managed app with deletion prevention
+
+Like for cluster apps, you can use the `--prevent-deletion` parameter also with the command `kubectl gs template app`.
+
+```sh
+kubectl gs template app \
+  --cluster-name=mycluster \
+  --organization=myorg \
+  --catalog=giantswarm \
+  --app-name=mycluster-ingress-nginx \
+  --name=ingress-nginx \
+  --version=3.9.1 \
+  --target-namespace=kube-system \
+  --prevent-deletion
+```
+
+## Protecting an existing object
+
+In this example, the object `App/myapp` and its related user values config map receive deletion prevention:
+
+```sh
+kubectl label -n org-ORGANIZATION App myapp giantswarm.io/prevent-deletion=true
+kubectl label -n org-ORGANIZATION ConfigMap myapp-userconfig giantswarm.io/prevent-deletion=true
+```
+
+Each label only protects a single object, not a hierarchy of objects such as seen with CAPI clusters (`Cluster`, `AWSCluster` and other children). For workload clusters and managed apps, please prefer the aforementioned instructions instead of this manual labeling.
+
+## Configure targeted resources {#targeted-resource-configuration}
+
+If the [list of targeted resources](https://github.com/giantswarm/kyverno-policies-ux/blob/main/helm/kyverno-policies-ux/values.yaml)
+does not cover all resources that you want to protect, it is possible to configure them through the user config values of the `kyverno-policies-ux` app.
+[Here]({{< relref "/tutorials/fleet-management/app-platform/app-configuration/" >}}) you can learn more about how to configure an app.
```

Comment:
```suggestion
[Here]({{< relref "/tutorials/fleet-management/app-platform/app-configuration/" >}}) you can learn more about how to configure an app.

```

Reply:
There's no difference apart from the extraneous newline. My editor is already correctly set to have a `\n` at end of file.